src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -222,6 +224,20 @@ public virtual async Task CallAsync(string method, Dictionary<string, object> ar
             await base.CallAsync(method, arguments, cancellationToken);
         }
 
+        internal static void AddLanguageWorkerChannelErrors(IFunctionDispatcher functionDispatcher, IDictionary<string, ICollection<string>> functionErrors)
+        {
+            foreach (KeyValuePair<WorkerConfig, LanguageWorkerState> kvp in functionDispatcher.LanguageWorkerChannelsState)
+            {
+                WorkerConfig workerConfig = kvp.Key;
+                LanguageWorkerState workerState = kvp.Value;
+                foreach (var functionRegistrationContext in workerState.RegisteredFunctions)
+                {
+                    var languageWorkerChannelException = new LanguageWorkerChannelException(workerState, $"Failed to start language worker process for: {workerConfig.Language}");
```

Comment:
Based on name, this method looks like it's designed to handle general channel errors, not just startup errors. However, this is a startup specific error message?

Reply:
Throwing this exception specifically when unable to start a language worker process.